lib/App/Config/Chronicle.pm

```diff
@@ -340,6 +346,10 @@ sub save_dynamic {
     my $self = shift;
     my $settings = $self->chronicle_reader->get($self->setting_namespace, $self->setting_name) || {};
 
+    my ($package, $filename, $line) = caller;
+    warn "Deprecated call used (save_dynamic). Called from package: $package | file: $filename | line: $line"
+        if ($self->_warn_for_deprecated && $package ne __PACKAGE__);
+
```

Comment:
Something like [`Mojo::Util::deprecated()`](https://metacpan.org/pod/Mojo::Util#deprecated) could be an alternative, but this is fine. :+1:

Reply:
Or just `warnings::warnif deprecated => ...` which uses the standard core warnings bitflags